src/GoTrueClient.ts

```diff
@@ -887,4 +892,33 @@ export default class GoTrueClient {
       console.error('_handleVisibilityChange', error)
     }
   }
+
+  /**
+   * Generates the relevant login URL for a third-party provider.
+   * @param provider One of the providers supported by GoTrue.
+   * @param options.redirectTo A URL or mobile address to send the user to after they are confirmed.
+   * @param options.scopes A space-separated list of scopes granted to the OAuth application.
+   * @param options.queryParams An object of key-value pairs containing query parameters granted to the OAuth application.
+   */
+  private _getUrlForProvider(
```

Comment:
@alaister made this private because i don't see a usecase where someone would want to get the `/authorize` endpoint separately

Reply:
Nice. We can always add it back if needed!